activation/post.go

```diff
@@ -63,9 +63,10 @@ type PostSetupManager struct {
 	lastOpts *atypes.PostSetupOpts
 	lastErr  error
 
-	// startedChan indicates whether a data creation session has started.
-	// The channel instance is replaced in the end of the session.
-	startedChan chan struct{}
+	// ctx and cancel are the context and cancel function of the initialization
+	// they can be used to stop and monitor a started session
+	ctx    context.Context
+	cancel context.CancelFunc
```

Comment:
What's the point of storing the context? Isn't it enough to story the `cancel` function only?
```suggestion
	cancel context.CancelFunc
```